message_adapter/message_adapter.py

```diff
@@ -429,7 +429,7 @@ def createNextEvent(self, handlerResponse, event, messageConfig):
         """
         self.__validate_json(handlerResponse, 'output')
         result = self.__assignOutputs(handlerResponse, event, messageConfig)
-        if 'exception' not in result:
+        if 'exception' not in result or (not result['exception']):
```

Comment:
We can use `dict.get()` method to simplify the code:
`if result.get('exception') is None:`
line 119:
`if exception != 'None' and event.get('exception', 'None') == 'None':`

Reply:
Good point 👍

Reply:
I updated it to use `dict.get()` but it's not foolproof because Python only checks for the existence of the key, not whether the value is `None`, so I'm still using `not` and a separate check for  the string `'None'`.

Reply:
👍